src/src.pro

```diff
@@ -7,7 +7,15 @@ QT -= gui
 
 CONFIG += c++11 hide_symbols link_pkgconfig
 PKGCONFIG += profile mlite5 mce timed-qt5 blkid libcrypto libsailfishkeyprovider connman-qt5 glib-2.0
-PKGCONFIG += ssu-sysinfo nemodbus packagekitqt5 libsystemd sailfishusermanager sailfishaccesscontrol
+PKGCONFIG += ssu-sysinfo nemodbus libsystemd sailfishusermanager sailfishaccesscontrol
+
+packagesExist(packagekitqt5) {
+    message("Developer mode plugin enabled")
+    PKGCONFIG += packagekitqt5
```

Comment:
Now this has both CONFIG and packagesExist for enabling this. Think best would be to just use explicit config. Maybe even opt-out instead of opt-in.